nexusdialog/src/main/java/com/github/dkharrat/nexusdialog/FormActivity.java

```diff
@@ -24,14 +24,24 @@ public void onCreate(Bundle savedInstanceState) {
         formController = new FormController(this);
         initForm();
 
+        if (savedInstanceState != null && savedInstanceState.containsKey(MODEL_KW)) {
+            FormModel savedModel = (FormModel) savedInstanceState.getSerializable(MODEL_KW);
+            formController.setModel(savedModel);
```

Comment:
The validation of the form does not store data but only process every fields, the validation errors display only rely on an interface which is provided on the FormController creation.
The only drawback will be that any validation display on the form's fields will be reset until next validation. (if the validation display method uses the form at all)

So I don't really think the library has to handle this.

Having said that, if you really want to save the state of the validation display, I think the simplest way is to add errors  management methods to the `FormModel` (something like a `getError(String fieldName)` and `setError(String fieldName)`).
That way every persisted data will be stored in the FormData.